File: CS/TutorialsCustomItems/CustomItems/HelloWorldItem.cs
using DevExpress.DashboardCommon;
using DevExpress.DashboardWin;
using DevExpress.XtraEditors;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace TutorialsCustomItems.CustomItems
{
    [DisplayName("Hello World Item"),
     CustomItemDescription("Hello World Description"),
     CustomItemImage("TutorialsCustomItems.images.HelloWorldIcon.svg")]
    class HelloWorldItemMetadata : CustomItemMetadata
    {

    }

    public class HelloWorldItemProvider : CustomControlProviderBase
    {
        MemoEdit textbox;
        protected override Control Control { get { return textbox; } }
        public HelloWorldItemProvider()
        {
            textbox = new MemoEdit();
            textbox.Text = "Hello World!";
            textbox.ReadOnly = true;
            textbox.ContextMenu = new ContextMenu();

        }
    }
}

File: VB/TutorialsCustomItems/CustomItems/HelloWorldItem.vb
Imports DevExpress.DashboardCommon
Imports DevExpress.DashboardWin
Imports DevExpress.XtraEditors
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace TutorialsCustomItems.CustomItems
    <DisplayName("Hello World Item"), CustomItemDescription("Hello World Description"), CustomItemImage("HelloWorldIcon.svg")>
    Friend Class HelloWorldItemMetadata
		Inherits CustomItemMetadata

	End Class

	Public Class HelloWorldItemProvider
		Inherits CustomControlProviderBase

		Private textbox As MemoEdit
		Protected Overrides ReadOnly Property Control() As Control
			Get
				Return textbox
			End Get
		End Property
		Public Sub New()
			textbox = New MemoEdit()
			textbox.Text = "Hello World!"
			textbox.ReadOnly = True
			textbox.ContextMenu = New ContextMenu()

		End Sub
	End Class
End Namespace
